Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,528 @@
+remote: #0 building with "default" instance using docker driver
+remote:
+remote: #1 [internal] load build definition from Dockerfile
+remote: #1 transferring dockerfile: 1.50kB done
+remote: #1 DONE 0.1s
+remote:
+remote: #2 [internal] load metadata for mcr.microsoft.com/dotnet/sdk:8.0
+remote: #2 DONE 0.2s
+remote:
+remote: #3 [internal] load metadata for docker.io/library/debian:latest
+remote: #3 DONE 0.4s
+remote:
+remote: #4 [internal] load .dockerignore
+remote: #4 transferring context: 51B done
+remote: #4 DONE 0.0s
+
+### Truncated for Brevity
+
+=====> Application deployed:
+       http://platform.dev.catenatools.com
+       https://platform.dev.catenatools.com
+
+To jlyon1.catenatools.com:platform
+ * [new branch]      HEAD -> main
+```
+
+Now we're all set! 
+
+> You can update your platform the same way, simply commit the changes and push them to the `main` branch on the dokku remote. The server will automatically build and deploy your changes with no downtime.
+
+
+You can test your platform by logging in:
+
+**Git Bash**
+```Bash
+SESSION_ID=$(curl -D - -sS --location '$CATENA_URL/api/v1/authentication/login' \
+      --header 'Content-Type: application/json' \
+      --data '{
+      "provider": "PROVIDER_UNSAFE",
+      "payload": "test53"
+  }' | grep "session-*" | cut -d' ' -f2)
+
+curl --location '$CATENA_URL/api/v1/accounts' \
+--header "session-id: $SESSION_ID" \
+--header 'Content-Type: application/json' \
+--data '{}'
+
+# {"account":{"id":"account-f378e3c2-8760-4059-ba91-69f5343dfb48","displayName":"test53","authRole":"user","metadata":{},"providers":["PROVIDER_UNSAFE"]}}
+```
+
+[//]: # (TODO: Add the link here.)
+The full postman collection can be found here. Set the http-host environment variable to your domain prior to use.
+
+## Running your game in a multiplayer configuration
+
+Next we will use the server [Sidecar](Sidecar.md) along with our backend to support a multiplayer workflow.
+
+> The sidecar enables you to configure your game to work with one API interface, and deploy it on multiple fleet managers. It can also make requests on behalf of the game server, enabling integrations which require zero server-side integration.
+
+We are making some assumptions and trade-offs to simplify this guide. There is an example implementation in our [catena-lyra-demo repo.](https://github.com/CatenaTools/catena-lyra-demo/tree/main)
+
+1. The server will be "always open" and accepting players
+2. There will be only one type of match
+
+### Setup our Game Server
+
+We will be running our game on an AWS EC2 Instance running Windows Server, these instructions also work with a Linux server using minor modifications.
+
+First create a key pair in the AWS console for remote access to the machine:
+
+![access_key_quickstart](access_key_quickstart.png)
+
+Next, configure terraform to provision an ec2 instance where we will run the server, this should be in `ec2/main.tf`
+
+```
+resource aws_instance "server_instance" {
```

Comment:
Carrying comment over from last PR. We should likely just provide this as another directory in the `infrastructure` repo rather than requiring copy/paste. I can work on that after I'm done with GameLift stuff.

Reply:
Sounds good, i've put it in a ticket here: https://catenatools.atlassian.net/browse/CAT-567

Reply:
Open PR here: https://github.com/CatenaTools/infrastructure/pull/9